src/DIRAC/FrameworkSystem/API/AuthHandler.py

```diff
@@ -450,10 +450,10 @@ def __researchDIRACGroup(self, extSession, chooseScope, state):
         ]
         if not validGroups:
             return None, self.server.handle_response(
-                getHTML(
+                payload=getHTML(
```

Comment:
Where is this used?

Reply:
if no group is specified when requesting authorization, the user will return HTML with a selection of groups